### PR TITLE
Update discoverModelDefinition description

### PR DESCRIPTION
**Pull request by dhmlau**
Original issue: https://github.com/strongloop-internal/scrum-apex/issues/127

### CODE REVIEW

**Comment by dhmlau** [comment]
@slnode  test please

**Comment by dhmlau** [comment]
@crandmck @strongloop/sq-lb-apex  PTAL

**Review by ssh24 — APPROVED** [review]

**Review by jannyHou — APPROVED** [review]
LGTM

**Review by b-admike — APPROVED** [review]

**Review by 0candy — APPROVED** [review]

File: pages/en/lb2/Discovering-models-from-relational-databases.md [hunk 1]
```diff
@@ -83,6 +83,9 @@ function (err, models) {
 Some connectors provide discovery capability so that we can use DataSource to discover model definitions from existing database schema.
 The following APIs enable UI or code to discover database schema definitions that can be used to build LoopBack models.
 
+`discoverModelDefinintions()` by default discovers database schema definitions owned by the userid used to connect to the database.
```
Comment (kjdelisle) [inline pages/en/lb2/Discovering-models-from-relational-databases.md]: Typo: `discoverModelDefinintions()` -> `discoverModelDefinitions()`

File: pages/en/lb2/Discovering-models-from-relational-databases.md [hunk 2]
```diff
@@ -83,6 +83,9 @@ function (err, models) {
 Some connectors provide discovery capability so that we can use DataSource to discover model definitions from existing database schema.
 The following APIs enable UI or code to discover database schema definitions that can be used to build LoopBack models.
 
+`discoverModelDefinintions()` by default discovers database schema definitions owned by the userid used to connect to the database.  
+Calling it with the `all: true` option makes the discovery include database schema definitions of all owners.
```
Comment (kjdelisle) [inline pages/en/lb2/Discovering-models-from-relational-databases.md]: `include database schema` -> `include the database schema`

File: pages/en/lb3/Discovering-models-from-relational-databases.md [hunk 3]
```diff
@@ -83,6 +83,9 @@ function (err, models) {
 Some connectors provide discovery capability so that we can use DataSource to discover model definitions from existing database schema.
 The following APIs enable UI or code to discover database schema definitions that can be used to build LoopBack models.
 
+`discoverModelDefinintions()` by default discovers database schema definitions owned by the userid used to connect to the database.
```
Comment (kjdelisle) [inline pages/en/lb3/Discovering-models-from-relational-databases.md]: Typo: `discoverModelDefinintions()` -> `discoverModelDefinitions()`

**Review by kjdelisle — CHANGES_REQUESTED** [review]
Kinda disappointed that this many people greenlit this PR when there are two obvious typos. 👎

**Comment by dhmlau** [comment]
sorry about the typo.  @kjdelisle  PTAL

**Review by crandmck — APPROVED** [review]

**Comment by crandmck** [comment]
Thanks!

**Pull request merged**